### PR TITLE
use pr icon to show if a pr is open or not

### PR DESCRIPTION
**Pull request by butlerx**
screen shot 2017-10-01 at 12 03 30 pm

### CODE REVIEW

**Comment by jenkoian** [comment]
Thanks for this, really like the idea of an icon next to the PR name. However, I do have some feedback:

- Move the icon to the left of the text 
- Colour the icon rather than the background (would be neat to have something similar to this: https://github.com/pulls)
- Try and use [tachyons](http://tachyons.io/docs/) utility classes rather than custom classes

**Comment by butlerx** [comment]
screen shot 2017-10-01 at 3 07 58 pm
ok im moved the icon and used a different icon for open pr. 
ive replaced css-grids with tachyons aswell

File: views/index.hbs [hunk 1]
```diff
@@ -1,4 +1,4 @@
-﻿<h1 class="f1 f-6-ns orange tc"><a class="orange no-underline" href="/">Hacktoberfest Checker</a></h1>
```
Comment (jenkoian) [inline views/index.hbs]: This causes the main logo to be left aligned so don't think we want to remove this one?

File: views/partials/prs.hbs [hunk 2]
```diff
@@ -8,8 +8,19 @@
...
+          </div>
+        {{else}}
+          <div class="fl w-10 pa2 closed">
+            <svg aria-hidden="true" height="24" version="1.1" viewBox="0 0 12 16" width="32"><path fill-rule="evenodd" d="M10 7c-.73 0-1.38.41-1.73 1.02V8C7.22 7.98 6 7.64 5.14 6.98c-.75-.58-1.5-1.61-1.89-2.44A1.993 1.993 0 0 0 2 .99C.89.99 0 1.89 0 3a2 2 0 0 0 1 1.72v6.56c-.59.35-1 .99-1 1.72 0 1.11.89 2 2 2a1.993 1.993 0 0 0 1-3.72V7.67c.67.7 1.44 1.27 2.3 1.69.86.42 2.03.63 2.97.64v-.02c.36.61 1 1.02 1.73 1.02 1.11 0 2-.89 2-2 0-1.11-.89-2-2-2zm-6.8 6c0 .66-.55 1.2-1.2 1.2-.65 0-1.2-.55-1.2-1.2 0-.65.55-1.2 1.2-1.2.65 0 1.2.55 1.2 1.2zM2 4.2C1.34 4.2.8 3.65.8 3c0-.65.55-1.2 1.2-1.2.65 0 1.2.55 1.2 1.2 0 .65-.55 1.2-1.2 1.2zm8 6c-.66 0-1.2-.55-1.2-1.2 0-.65.55-1.2 1.2-1.2.65 0 1.2.55 1.2 1.2 0 .65-.55 1.2-1.2 1.2z"></path></svg>
+          </div>
+        {{/if}}
+        <div class"fl w-90 pa2">
```
Comment (jenkoian) [inline views/partials/prs.hbs]: Missing a `=` here.

**Comment by jenkoian** [comment]
Thanks @butlerx !

**Pull request merged**